### PR TITLE
tko_web_sessiones_management improvements

### PR DESCRIPTION
**Pull request by acysos**
Hello,

Add spanish translation, translation support for login error messages, show specific login error reason to the user instead all reason.

Greetings

### CODE REVIEW

File: tko_web_sessions_management/main.py [hunk 1]
```diff
@@ -75,6 +75,9 @@ def web_login(self, redirect=None, **kw):
             if 'login' in request.params and 'password' in request.params:
                 uid = request.session.authenticate(request.session.db, request.params[
                                                    'login'], request.params['password'])
+            reason1 = False
+            reason2 = False
+            reason3 = False
```
Comment (cmsalmeida) [inline tko_web_sessions_management/main.py]: Please don't say to user why he can't access that is a security breach.
By displaying exactly what is wrong with the login attempt, you are helping an attacker narrow down his attack. We should give always a generic message.
Although we log the reason in user attempts so admin or support can explain or fix the problem.

Thank you for your PR!

File: tko_web_sessions_management/main.py [hunk 2]
```diff
@@ -133,18 +136,22 @@ def web_login(self, redirect=None, **kw):
                                         calendar_group = group.name
                                 if sessions and group.multiple_sessions_block and multi_ok:
                                     multi_ok = False
-                                    unsuccessful_message = "unsuccessful login from '%s', multisessions block defined in group '%s'" % (
+                                    unsuccessful_message = _("unsuccessful login from '%s', multisessions block defined in group '%s'") % (
```
Comment (cmsalmeida) [inline tko_web_sessions_management/main.py:136]: :+1:

File: tko_web_sessions_management/main.py [hunk 3]
```diff
@@ -133,18 +136,22 @@ def web_login(self, redirect=None, **kw):
...
                                         request.params['login'], group.name)
+                                    reason2 = True
                                     break
                             if calendar_set > 0 and calendar_ok == False:
-                                unsuccessful_message = "unsuccessful login from '%s', user time out of allowed calendar defined in group '%s'" % (
+                                unsuccessful_message = _("unsuccessful login from '%s', user time out of allowed calendar defined in group '%s'") % (
```
Comment (cmsalmeida) [inline tko_web_sessions_management/main.py:140]: :+1:

File: tko_web_sessions_management/main.py [hunk 4]
```diff
@@ -133,18 +136,22 @@ def web_login(self, redirect=None, **kw):
...
                                     request.params['login'], calendar_group)
+                                reason3 = True
                     else:
-                        unsuccessful_message = "unsuccessful login from '%s', multisessions block defined in user" % request.params[
+                        unsuccessful_message = _("unsuccessful login from '%s', multisessions block defined in user") % request.params[
```
Comment (cmsalmeida) [inline tko_web_sessions_management/main.py:143]: :+1:

File: tko_web_sessions_management/main.py [hunk 5]
```diff
@@ -133,18 +136,22 @@ def web_login(self, redirect=None, **kw):
...
                             'login']
+                        reason2 = True
             else:
-                unsuccessful_message = "unsuccessful login from '%s', wrong username or password" % request.params[
+                unsuccessful_message = _("unsuccessful login from '%s', wrong username or password") % request.params[
```
Comment (cmsalmeida) [inline tko_web_sessions_management/main.py:146]: :+1:

File: tko_web_sessions_management/main.py [hunk 6]
```diff
@@ -164,11 +171,14 @@ def web_login(self, redirect=None, **kw):
...
-            values[
-                'reason3'] = '- User not allowed to login at this specific time or day'
+            values['error'] = _('Login failed due to one of the following reasons:')
+            if reason1:
+                values['reason1'] = _('- Wrong login/password')
+            if reason2:
+                values['reason2'] = _('- User not allowed to have multiple logins')
+            if reason3:
+                values[
+                    'reason3'] = _('- User not allowed to login at this specific time or day')
```
Comment (cmsalmeida) [inline tko_web_sessions_management/main.py]: please leave this as generic message as explained above.
just let us give the possible reasons...

**Comment by cmsalmeida** [comment]
Fix the reason message, to be generic (stating all possible reasons) please, I'll merge asap.
Good contribution thanks

File: tko_web_sessions_management/main.py (hunk 1 above)
Comment (acysos) [inline tko_web_sessions_management/main.py]: Yes, you have some reason. But don't tell the user what happen with generate support tickets.

I have remove it.

**Comment by acysos** [comment]
I have returned the reason to the original code.

Greetings

**Comment by cmsalmeida** [comment]
:+1:

**Pull request merged**